File: java/example1/src/main/java/example1/example_1.java
package example1;

import com.fusioncharts.fusionexport.client.*; // import sdk

public class example_1 {
public static void main(String[] args) throws Exception {
	String configPath = System.getProperty("user.dir") + "/resources/template.html";

        // Instantiate the ExportConfig class and add the required configurations
        ExportConfig config = new ExportConfig();
        // Provide path of the chart configuration which we have defined above.
        // You can also pass the same object as serialized JSON.
        config.set("templateFilePath", configPath);

        // ++++++ ATTENTION - MODIFY THE EXPORT TYPE ++++++
        // OPTIONS ARE: 'png' (default) | 'jpeg' | 'pdf' | 'svg' | 'html' | 'csv' | 'xls' | 'xlsx'
        config.set("type", "pdf");
        config.set("templateFormat", "A4");
       
        config.set("asyncCapture", true);

        // Instantiate the ExportManager class
        ExportManager manager = new ExportManager();
        // Call the export() method with the export config and the respective callbacks
        manager.export(config, ".",  true);

}
}

File: PHP/example1/example_1.php
<?php
  // Import dependencies
  require DIR__ . '/../vendor/autoload.php';
  use FusionExport\ExportManager;
  use FusionExport\ExportConfig;

// Instantiate the ExportConfig class and add the required configurations
$exportConfig = new ExportConfig();
  // Provide path of the chart configuration which we have defined above.  // You can also pass the same object as serialized JSON.
  $exportConfig->set('templateFilePath', realpath('resources/template.html'));

// ++++++ ATTENTION - MODIFY THE EXPORT TYPE ++++++
// OPTIONS ARE: 'png' (default) | 'jpeg' | 'pdf' | 'svg' | 'html' | 'csv' | 'xls' | 'xlsx'
$exportConfig->set('type', 'pdf');
$exportConfig->set('templateFormat', 'A4');
$exportConfig->set('asyncCapture', true);

// Instantiate the ExportManager class
$exportManager = new ExportManager();

// Call the export() method with the exportConfig and the respective callbacks
$exportManager->export($exportConfig, $outputDir = '.', $unzip = true);
?>